### DISCUSSION TITLE
Training for Github Student Expert Program

### DISCUSSION THREAD

**Discussion by pushkar-web** [community/community#146512] [GitHub Education] [answered]
### Hi everyone,

I have received a mail about claiming the student developer pack.but i am not able to start the training for the github student expert program as mentioned in the offers about Github Campus Expert training for the qualified participants.
![image](https://github.com/user-attachments/assets/88adadf2-a1a0-45f5-9b82-337e2330e214)

**Comment by davevad93** [comment, marked as answer]
Hi @pushkar-web ,

If you mean [Applying to the GitHub Campus Experts program](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/applying-to-be-a-github-campus-expert#applying-to-the-github-campus-experts-program), for now applications are closed, they open in February and August, so you have to wait a couple of months to apply.
